### ISSUE TITLE
False Positiv PSR12.Operators.OperatorSpacing.NoSpaceAfter

### ISSUE THREAD

**Issue by puggan** [squizlabs/PHP_CodeSniffer#3180]
**Describe the bug**
Using "-" or "+" to force an string to a numeric value, sometimes triggers PSR12.Operators.OperatorSpacing.NoSpaceAfter.
In the exemple bellow it's triggered on the concatination rows, but not the other 2 rows.

**Code sample**
```php
$string = '3';
echo 'Pos: ', +$string, PHP_EOL; // ok
echo 'Neg: ', -$string, PHP_EOL; // ok
echo 'Pos: ' . +$string . PHP_EOL; // PSR12.Operators.OperatorSpacing.NoSpaceAfter
echo 'Neg: ' . -$string . PHP_EOL; // PSR12.Operators.OperatorSpacing.NoSpaceAfter
```

**Custom ruleset**
`--standard=PSR12`

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs --standard=PSR12 test.php`
3. See error message displayed
```
FILE: /..../test.php
----------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------
 6 | ERROR | [x] Expected at least 1 space after "+"; 0 found
 7 | ERROR | [x] Expected at least 1 space after "-"; 0 found
----------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------
```

**Expected behavior**
consistent.
Trigger on zero or all four of the rows,
I myself prefer if zero of the above rows trigger.

**Versions (please complete the following information):**
 - OS: Ubuntu 20.10
 - PHP: 7.4.9
 - PHPCS: 3.5.8
 - Standard: PSR12

**Additional context**
Looks related to  #2696

**Comment by jrfnl** [comment]
Duplicate of #3170, a patch for this has already been merged and will be included in the PHPCS 3.6.0 release.
Suggest: close.

**Comment by puggan** [comment]
The added test seams to only regard the "-" version, but I assume the fix also fixes the "+" version.
Maybe there should be test for the "+" version too.

**Comment by jrfnl** [comment]
The logic is no different for `+` or `-`, so no additional test is needed.
